### DISCUSSION TITLE
Welcome to aoc-2020 Discussions!

### DISCUSSION THREAD

**Discussion by rdmueller** [docToolchain/aoc-2020#135] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by t-pi** [comment]
Hey, the brand new discussions are here! Greaty, another touchpoint tool ;)
However, I just wanted to test, will stick to Slack for the rest of AoC-2020 interactions..

**Reply by rdmueller** [reply]
yep. I also just wanted to give it a try...

**Reply by rdmueller** [reply]
hm. somehow I don't see the sense in these discussions... the issues are quite nice...

**Reply by t-pi** [reply]
Well, lots of 'issues' are rather discussions. I guess with tags & filters these two things could be reintegrated again...
